src/components/config/validate.ts

```diff
@@ -320,6 +320,17 @@ export const validateConfig = (configuration: Config): Validation => {
 
     if ((!socket && (requests?.length ?? 0)) === 0) return PROBE_NO_REQUESTS
 
+    // Validate Interval
+    const timeouts = requests.map((req) => req.timeout)
+    const totalTimeout = timeouts.reduce((a, b) => a + b)
+    const totalTimeoutSeconds = totalTimeout / 1000
+
+    if (totalTimeoutSeconds > interval) {
```

Comment:
Also need to make sure each timeout is greater than 0

Reply:
Updated!